jsapp/js/components/formLanding/formLanding.js

```diff
@@ -233,23 +234,26 @@ class FormLanding extends React.Component {
                 return (
                   <bem.FormView__group m='items' key={n} className={n >= this.state.DVCOUNT_LIMIT ? 'hidden' : ''}>
                     <bem.FormView__label m='version'>
-                      {`v${dvcount - n}`}
+                      <span>{`v${dvcount - n}`}</span>
                       {item.uid === this.state.deployed_version_id && this.state.deployment__active && (
-                        <bem.FormView__cell m='deployed'>{t('Deployed')}</bem.FormView__cell>
```

Comment:
I'm guessing after this change we can remove `.form-view__cell--deployed` styles?